app/controllers/courses_controller.rb

```diff
@@ -9,7 +9,6 @@ class CoursesController < ApplicationController
 
   def edit
     I18n.locale = @course.locale || I18n.default_locale
-    cookies[:edited_course] = params[:id]
```

Comment:
Why is this line removed?

Reply:
I did not see any place in the codebase where this cookie is actually used (anymore).